### PR TITLE
golint fixes (no functional changes)

### PR DESCRIPTION
**Pull request by gumpt**
Just trying to learn a bit about the codebase, picked #153 out of the issue list because fixing linter warnings is simple.

Let me know what I could do better. :-)

### CODE REVIEW

**Comment by ianvernon** [comment]
Thanks for contributing to Cilium! :) 
Can you please amend your commits with a sign-off message? Refer to http://docs.cilium.io/en/stable/contributing/#developer-s-certificate-of-origin for more information. Thanks!

**Comment by gumpt** [comment]
Fixed, should've done that to start.

File: daemon/endpoint.go [hunk 1]
```diff
@@ -62,34 +62,34 @@ func (h *getEndpoint) Handle(params GetEndpointParams) middleware.Responder {
...
+	wg.Add(len(endpointmanager.Endpoints))
+	for k := range endpointmanager.Endpoints {
+		go func(wg *sync.WaitGroup, ep *endpoint.Endpoint) {
+			ep.Mutex.RLock()
+			if ep.HasLabels(convertedLabels) {
+				eps = append(eps, ep.GetModel())
+			}
+			ep.Mutex.RUnlock()
+			wg.Done()
+			}(&wg, endpointmanager.Endpoints[k])
```
Comment (aanm) [inline daemon/endpoint.go]: Can you run `gofmt -w daemon/endpoint.go`? The verifier is failing because of this line.

**Review by aanm — CHANGES_REQUESTED** [review]
There's no need for the `Merge branch 'master' into master` commit ;-)

File: daemon/endpoint.go (hunk 1 above)
Comment (gumpt) [reply daemon/endpoint.go]: Rebased with exec to run gofmt.

**Review by aanm — APPROVED** [review]

**Review by ianvernon — APPROVED** [review]

**Comment by tgraf** [comment]
Awesome, thanks @gumpt!

**Pull request merged**